src/status_im/hardwallet/fx.cljs

```diff
@@ -95,3 +98,32 @@
  :send-transaction-with-signature
  (fn [{:keys [transaction signature on-completed]}]
    (status/send-transaction-with-signature transaction signature on-completed)))
+
+(re-frame/reg-fx
+ :hardwallet/persist-pairings
+ (fn [pairings]
+   (.. js-dependencies/async-storage
+       (setItem "status-keycard-pairings" (types/clj->json pairings)))))
```

Comment:
why not use transit here?

Reply:
Didn't though about transit, does it make sense here?

Reply:
yes it is faster and there is already the functions to do it in the same types namespace you used here